### ISSUE TITLE
[Bug]: httpv2 400 Bad request

### ISSUE THREAD

**Issue by nikodevv** [Amplitude-Developer-Docs/amplitude-dev-center#188]
### Contact Details

niko@productwind.com

### What happened?

We have integrated and use the amplitude SDK with no issues. We want to move some of our events server side, and we plan to do this with the [httpv2 api](https://www.docs.developers.amplitude.com/analytics/apis/http-v2-api). However our requests fail with the following response:

```
<html>
<head><title>400 Bad Request</title></head>
<body>
<center><h1>400 Bad Request</h1></center>
</body>
</html>
```

Our request uses the [httpv2 postman template](https://www.postman.com/amplitude-developer-docs/workspace/amplitude-developers/request/20044411-25cbda01-383c-453c-a726-7589c0a97580) provided by amplitude, with only 3 properties changed (`api_key`, `user_id`, and `devce_id`). This happens in both our master and production environments, with no additional data provided. We suspect the issue is related to our starter-plan API keys not having sufficient permissions. We copied the api_keys from `https://analytics.amplitude.com/XXX/settings/projects/XXX/general`.

We also noticed that the endpoint urls in the postman request are different from the ones in the documentation page (`https://api2.amplitude.com/2/httpapi` vs `https://api.amplitude.com/2/httpapi`). We have tried both urls.

Request Example:

Method: POST
Headers:
```
Content-Type: application/json
```
Body:

```
{
  "api_key": "REDACTED",
  "events": [
    {
      "user_id": "REDACTED",
      "device_id": "REDACTED",
      "event_type": "watch_tutorial",
      "time": 1396381378123,
      "event_properties": {
        "load_time": 0.8371,
        "source": "notification",
        "dates": [
          "monday",
          "tuesday"
        ]
      },
      "user_properties": {
        "age": 25,
        "gender": "female",
        "interests": [
          "chess",
          "football",
          "music"
        ]
      },
      "groups": {
        "team_id": "1",
        "company_name": [
          "Amplitude",
          "DataMonster"
        ]
      },
      "app_version": "2.1.3",
      "platform": "iOS",
      "os_name": "Android",
      "os_version": "4.2.2",
      "device_brand": "Verizon",
      "device_manufacturer": "Apple",
      "device_model": "iPhone 9,1",
      "carrier": "Verizon",
      "country": "United States",
      "region": "California",
      "city": "San Francisco",
      "dma": "San Francisco-Oakland-San Jose, CA",
      "language": "English",
      "price": 4.99,
      "quantity": 3,
      "revenue": -1.99,
      "productId": "Google Pay Store Product Id",
      "revenueType": "Refund",
      "location_lat": 37.77,
      "location_lng": -122.39,
      "ip": "127.0.0.1",
      "idfa": "AEBE52E7-03EE-455A-B3C4-E57283966239",
      "idfv": "BCCE52E7-03EE-321A-B3D4-E57123966239",
      "adid": "AEBE52E7-03EE-455A-B3C4-E57283966239",
      "android_id": "BCCE52E7-03EE-321A-B3D4-E57123966239",
      "event_id": 23,
      "session_id": 1396381378123,
      "insert_id": "5f0adeff-6668-4427-8d02-57d803a2b841"
    }
  ]
}
```

**Comment by caseyamp** [comment]
Hi there @nikodevv! Thanks for reaching out. This is the repo for the developer docs - we aren't staffed to help troubleshoot issues with Amplitude itself, **but** you can get help by posting this question to the Amplitude Community.  Check out the community [here](https://community.amplitude.com/using-amplitude-31).

**Comment by nikodevv** [comment]
@caseyamp I think the documentation regarding http v2 api requests is inaccurate. Would it be possible for another person to try the api call I posted above (that request is straight from docs)?

**Comment by caseyamp** [comment]
@nikodevv I've tried both the postman collection and the upload request body from the docs without changing anything, and both times I got a 200 response. That's the extent of troubleshooting I can help with. I highly recommend posting this issue to the [Amplitude Community](https://community.amplitude.com) -- there are lots of experienced folks over there who can help you get to the bottom of this issue.
